joomla/joomla-cms
# [3.8] Tweak update percentage message to include %-sign

Pull Request for Issue # .

### Summary of Changes

Adds a % character to the end of the progress bar Percent-Progress commentary during a Joomla version upgrade.

### Testing Instructions

Install Joomla previous to current version. Apply fix. Upgrade to current version. During upgrade the Percent-complete line below the progress bar should now end with a '%' character.

### Documentation Changes Required
None.
